build/docker/agent/Dockerfile

```diff
@@ -163,7 +161,7 @@ COPY --chown=$USERNAME:$USERNAME ./code/requirements.txt /workspace/
 ENV TORCH_CUDA_ARCH_LIST="5.2 6.0 6.1 7.0 7.5 8.0 8.6+PTX"
 ENV IABN_FORCE_CUDA=1
 
-RUN source ~/.bashrc && cat /workspace/requirements.txt | xargs -n 1 -L 1 pip install
+RUN source ~/.bashrc && pip install torch==1.13.1 && pip install -r /workspace/requirements.txt
```

Comment:
Why did you make this change?

Reply:
Local building failed with xargs without providing an error message, it worked again with pip install.
The need for the xargs variant was due to the git package 'detectron' which needs torch to be installed first (pip install doesn't care about the order of the requirements.txt).
Since we plan to replace the EfficientPS and most Perception things, detectron and other requirements will probably be unused soon anyway. But I can try it again with xargs since it used to work.

Reply:
No, I think your solution is fine. I was mainly curious, and your reasoning wasn't obvious to me.